### PR TITLE
added name attribute to input element

### PR DESCRIPTION
**Pull request by decisa**
at first I thought "why don't we have name attribute? let me add it anyways". now that I watched the solution videos, it should be there for sure ;)

### CODE REVIEW

**Comment by decisa** [comment]
updated the task from Kody the Koala to better reflect the video solution

File: src/exercise/06.js [hunk 1]
```diff
@@ -16,15 +16,16 @@ function UsernameForm({onSubmitUsername}) {
...
+  // 🐨 make sure to associate the label to the input. 
+  // to do so, set the value of 'htmlFor' prop of the label to the id of input
   return (
     <form>
       <div>
         <label>Username:</label>
         <input type="text" />
       </div>
-      <button type="submit">Submit</button>
+      <button name="username" type="submit">Submit</button>
```
Comment (kentcdodds) [inline src/exercise/06.js:28]: Did you mean to put this on the input?

**Review by decisa — COMMENTED** [review]
oh my :) my bad .. fixed it 👍

**Review by kentcdodds — APPROVED** [review]
Great! Thanks 👍

**Comment by kentcdodds** [comment]
@all-contributors please add @decisa for code

**Pull request merged**